fedimint-core/src/db/mod.rs

```diff
@@ -455,6 +457,9 @@ impl Database {
                     }
                 }
             }
+            let delay = (2u64.pow(curr_attempts.min(7) as u32) * 10).min(1000);
```

Comment:
The second `min` seems a bit redundant given that the max delay would still be 1280ms without it.